api/src/test/java/org/pmiops/workbench/api/CloudTaskUserControllerTest.java

```diff
@@ -121,13 +120,15 @@ public void testSynchronizeAccess() {
 
   @Test
   public void testCheckAndAlertFreeTierBillingUsage() {
-    Map<String, Double> googleBQCost = new HashMap<String, Double>();
-    googleBQCost.put("googleProject", 1.2);
-    UserBQCost userBQCost = new UserBQCost().userId(1L).workspaceBQCost(googleBQCost);
-    UserWorkspaceBQCostRequest request =
-        new UserWorkspaceBQCostRequest().userCostList(Arrays.asList(userBQCost));
-    controller.checkAndAlertFreeTierBillingUsage(request);
-    verify(mockFreeTierBillingUpdateService)
-        .checkAndAlertFreeTierBillingUsage(request.getUserCostList());
+    List<Long> userIdList = new ArrayList<Long>(Arrays.asList(1L, 2L, 3L));
+    controller.checkAndAlertFreeTierBillingUsage(userIdList);
+    verify(mockFreeTierBillingUpdateService).checkAndAlertFreeTierBillingUsage(userIdList);
+  }
+
+  @Test
+  public void testCheckAndAlertFreeTierBillingUsage_noUserListPassedFromTask() {
```

Comment:
why would this happen?

Reply:
What if the Task itself is corrupted, just want to make sure our task controller will be able to handle that